templates/nocodb/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.8"
+services:
+  nocodb:
+    image: nocodb/nocodb:latest
```

Comment:
can you please use a defined version? this in order to break the application in the future

Reply:
updated